packages/ui/.env.example

```
@@ -2,6 +2,6 @@ NEXT_PUBLIC_APP_NAME="Sui AMM"
 NEXT_PUBLIC_APP_DESCRIPTION="Full-Stack Sui AMM"
 NEXT_PUBLIC_LOCALNET_CONTRACT_PACKAGE_ID=""
 NEXT_PUBLIC_TESTNET_CONTRACT_PACKAGE_ID=""
-NEXT_PUBLIC_LOCALNET_AMM_CONFIG_ID=""
-NEXT_PUBLIC_TESTNET_AMM_CONFIG_ID=""
-NEXT_PUBLIC_LOCALNET_DEEPBOOK_REGISTRY_ID=""
\ No newline at end of file
+NEXT_PUBLIC_LOCALNET_DEEPBOOK_REGISTRY_ID=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)